battlebots/database/models.py

```diff
@@ -152,7 +152,7 @@ def loss_percentage(self):
 
     @property
     def rank(self):
-        bots = enumerate(session.query(Bot).order_by(desc(Bot.score)).all())
```

Comment:
Same as https://github.com/ZeusWPI/aichallenge/pull/85#discussion_r58984458, but this is a hard one as this is called from the ranker as well as from the webapp.